build.py

```diff
@@ -96,10 +96,14 @@ def __init__(self, name, env):
     def add_run_test_builds(self):
         # For everything that gets installed to build/unittests, add a rule for +basename
         # that runs the test from its original location.
+        unittest_dir = 'build/unittests/'
+        if self.globalEnv.TargetOSIs('windows'):
+            unittest_dir = unittest_dir.replace("/", "\\")
+
         tests = [flatten(build['inputs'])[0]
                  for build in self.builds
                  if build['rule'] == 'INSTALL'
-                 and flatten(build['outputs'])[0].startswith('build/unittests/')]
+                 and flatten(build['outputs'])[0].startswith(unittest_dir)]
         self.builds += [dict(outputs='+'+os.path.basename(test), inputs=test, rule='RUN_TEST')
```

Comment:
How do you feel about doing `outputs='+'+os.path.splitext(os.path.basename(test))[0] ` here? That would let you use the same +name_of_test on both *nix and windows without needing to add `.exe`. I'll leave it up to you.